pbs_light.go

```diff
@@ -793,24 +797,6 @@ func main() {
 	}
 }
 
-func setupExchanges(cfg *config.Configuration) {
-	exchanges = newExchangeMap(cfg)
```

Comment:
Now that these have been removed, there's a _lot_ of dead code which can be deleted from this file.

Start out with `makeExchangeMetrics` and try out the "find all references" feature in Visual Studio Code. You'll find a bunch of stuff which you can get rid of.

Reply:
Also ripped out the old metrics globals which uncovered a couple of other things. `makeExchangeMetrics` was only used twice however.